nuocd/nuodb_threads.py

```diff
@@ -36,9 +36,8 @@ def signal_handler(sig, frame):
     _pids = None
     try:
         # assume only one process by given name
-        _pid = subprocess.check_output(["pidof", process_name])
-        # _pid = subprocess.check_output(["pgrep", process_name ])
-        _pids = _pid[:-1].split()
```

Comment:
I think this was stripping away the trailing newline, but `str.split()` already does that.